Persephone/Views/AlbumCoverButton.swift

```diff
@@ -0,0 +1,43 @@
+//
```

Comment:
Dragging does not activate immediately, so we need to let the mouse move a little while still registering a click.